### DISCUSSION TITLE
透视表如果同时设置valueInCols:false和 hierarchyType:'tree'时总计行的数据无法展示出来

### DISCUSSION THREAD

**Discussion by MuRui666** [antvis/S2#3004] [Q&A]
如下图所示，当同时设置了valueInCols:false和 hierarchyType:'tree'时总计行的数据无法展示出来
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/4245c5de-7830-4220-a875-314287fcb2c4">

**Comment by lijinke666** [comment]
汇总 + 树状的场景下, 目前只支持 valueInCols: true

**Reply by MuRui666** [reply]
好的谢谢
